scripts/project/build_all_components.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# Build all components in a namespace (refer https://viash.io/reference/cli/ns_build.html)
+viash ns test --parallel
```

Comment:
the file name says build but the  cmd is for test? not sure if `test` is wrong or the filename is wrong

Reply:
Thanks for noticing!